.gitignore

```diff
@@ -2,3 +2,4 @@
 lib
 node_modules
 coverage
+.envrc
```

Comment:
Not a big deal, but do we need this line?

Reply:
Not really, I just ignore it in any project I touch that has local dev secrets. It's just because I use https://direnv.net/